### PR TITLE
MT3-425 #ready-for-test Add Review summary info to Review page

### PR DESCRIPTION
**Pull request by Gweaton**
Also updates anti-social behaviour question to use new version.

### CODE REVIEW

File: pages/thc/[processRef]/review.tsx [hunk 1]
```diff
@@ -112,12 +163,52 @@ const ReviewPage: NextPage = () => {
...
+            key: "Tenants",
+            value: tenantNames ? tenantNames.join(", ") : "loading",
+          },
+          {
+            key: "Tenure type",
+            value: tenureType.result ? tenureType.result : "loading",
+          },
+          { key: "Ouside Property:", value: outsidePropertyImageThumbnails },
+        ]}
+      />
```
Comment (erbridge) [inline pages/thc/[processRef]/review.tsx:199]: Could we reuse `TenancySummary` instead of recreating it?

File: pages/thc/[processRef]/review.tsx [hunk 2]
```diff
@@ -112,12 +163,52 @@ const ReviewPage: NextPage = () => {
...
+      {outsidePropertyImageThumbnails.length > 0 && (
+        <style jsx>{`
+          .mat-tenancy-summary img {
+            margin-right: 2em;
+          }
+        `}</style>
+      )}
+      <Paragraph>
+        The Tenancy and Household Check has now been completed. Please review
+        answers with tenant(s) and ask them to sign.
```
Comment (erbridge) [inline pages/thc/[processRef]/review.tsx:209]:
```suggestion
        the answers with all present tenants and ask them to sign.
```

I know it differs from the design, but this is clearer, IMO.

File: pages/thc/[processRef]/review.tsx [hunk 3]
```diff
@@ -112,12 +163,52 @@ const ReviewPage: NextPage = () => {
...
+          { key: "Ouside Property:", value: outsidePropertyImageThumbnails },
+        ]}
+      />
+      {outsidePropertyImageThumbnails.length > 0 && (
+        <style jsx>{`
+          .mat-tenancy-summary img {
+            margin-right: 2em;
+          }
+        `}</style>
+      )}
```
Comment (erbridge) [inline pages/thc/[processRef]/review.tsx:206]: Roll this into the value, I think, by creating a fragment component containing it and the thumbnails.

File: pages/thc/[processRef]/review.tsx [hunk 4]
```diff
@@ -112,12 +163,52 @@ const ReviewPage: NextPage = () => {
...
+      <Paragraph>
+        The Tenancy and Household Check has now been completed. Please review
+        answers with tenant(s) and ask them to sign.
+      </Paragraph>
+      <Paragraph>Present for check: {tenantsPresent}</Paragraph>
```
Comment (erbridge) [inline pages/thc/[processRef]/review.tsx:211]: Does this render correctly for multiple tenants?

**Review by erbridge — COMMENTED** [review]
Looks sensible as an approach. Some small comments.

File: pages/thc/[processRef]/review.tsx (hunk 4 above)
Comment (Gweaton) [reply pages/thc/[processRef]/review.tsx:211]: It does on my local version with 2 present tenants - is there something I should watch out for?

Comment (erbridge) [reply pages/thc/[processRef]/review.tsx:211]: Just wasn't sure what would happen with an array of strings (which is what I think this is). Does it join them with a comma?

File: pages/thc/[processRef]/review.tsx (hunk 1 above)
Comment (Gweaton) [reply pages/thc/[processRef]/review.tsx]: As discussed, this is less trivial than we thought - leaving for now.

File: pages/thc/[processRef]/review.tsx (hunk 4 above)
Comment (Gweaton) [reply pages/thc/[processRef]/review.tsx]: yep

Comment (Gweaton) [reply pages/thc/[processRef]/review.tsx]: I've actually changed this as I feel doing the `join` in that method feels like a bit too much in terms of SRP

**Review by erbridge — APPROVED** [review]
Looks good

**Pull request merged**